ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -338,58 +362,96 @@ runWith RunNodeArgs{..} LowLevelRunNodeArgs{..} =
       -> NodeToClientVersion
       -> NTC.Apps m (ConnectionId addrNTC) ByteString ByteString ByteString ()
     mkNodeToClientApps nodeKernelArgs nodeKernel blockVersion networkVersion =
-        NTC.mkApps
-          nodeKernel
-          rnTraceNTC
-          (NTC.defaultCodecs codecConfig blockVersion networkVersion)
-          (NTC.mkHandlers nodeKernelArgs nodeKernel)
+      NTC.mkApps
+        nodeKernel
+        rnTraceNTC
+        (NTC.defaultCodecs codecConfig blockVersion networkVersion)
+        (NTC.mkHandlers nodeKernelArgs nodeKernel)
 
     mkDiffusionApplications
-      :: MiniProtocolParameters
-      -> (   BlockNodeToNodeVersion blk
-          -> NTN.Apps m (ConnectionId addrNTN) ByteString ByteString ByteString ByteString ByteString ()
-         )
-      -> (   BlockNodeToClientVersion blk
+      :: NetworkP2PMode
+      -> MiniProtocolParameters
+      -> (BlockNodeToNodeVersion blk
+          -> NTN.Apps
+               m
+               (ConnectionId ntnAddr)
+               ByteString
+               ByteString
+               ByteString
+               ByteString
+               ByteString
+               ())
+      -> (BlockNodeToClientVersion blk
```

Comment:
Our style (link for reference https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus/docs/StyleGuide.md ) is

```
     Foo
  -> Bar
  -> Baz
```

so please re-introduce the leading space on the first argument.

Reply:
Please change `(BlockNodeToNodeVersion blk` to `(   BlockNodeToNodeVersion blk`